tests/test_integration.py

```diff
@@ -0,0 +1,93 @@
+import os
+import shutil
+import subprocess as sp
+import unittest
+
+__author__ = 'swenson'
+
+
+class LanguageIntegrationTests(unittest.TestCase):
```

Comment:
These could be broken out by language. Future testing will likely cover more output feature of the mappers beyond a simple string match. We'll likely want to be testing for child output, some formatting on type output, etc.